### DISCUSSION TITLE
Feature: Add a button "Add all Games"

### DISCUSSION THREAD

**Discussion by eaxxor** [Ni1kko/HourBoostr#6] [Ideas]
Hi,

Would it be possible to implement a button that adds all the games to the list to farm hours, having a ton of games is a pain in the rear to hold down Space to add them one by one in stead of clicking on them.

**Comment by Ni1kko** [comment]
So if i understand correctly you would like me to make multi select listbox. 
I can also add a new button to select random 32 games.
Adding all you can only idle 32 max.

Can do it today just getting food then ill do it.

**Comment by Ni1kko** [comment]
can i ask is it singleBoostr or hourBoostr? or both?

**Comment by eaxxor** [comment]
Hourboostr 
When you select games in lower right side, you get the option to search for your profile, after it fetched the games, a button to "add all games" would be great

**Comment by Ni1kko** [comment]
I can make it add random 32 games it wont track any more than 32 game at one time. i have done singleBoostr im going to do hourBoostr now

**Comment by Ni1kko** [comment]
https://github.com/Ni1kko/HourBoostr/releases/tag/4.2.0

**Comment by Ni1kko** [comment]
i still need to do the multi select list-box keep an eye out for that in near future update
